install/README.md

```diff
@@ -174,6 +179,56 @@ minikube start --kubernetes-version v1.10.0 --vm-driver virtualbox \
 
 > the --bootstrapper=localkube is required since we aren't using the `default` profile. ([bug](https://github.com/kubernetes/minikube/issues/2717))
 
+# Setting up an Amazon Web Services EKS cluster
+
+## Create EKS Instance
+
+Create your EKS instance using the [Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
+
+## Ensure VPC CNI 1.2 is Running
+
+EKS does not use the normal Kubernetes networking since it is [incompatible with Amazon VPC networking](https://www.contino.io/insights/kubernetes-is-hard-why-eks-makes-it-easier-for-network-and-security-architects). 
+
+In a console, run this command to get your current cni version
+
+```bash
+kubectl describe daemonset aws-node --namespace kube-system | grep Image | cut -d "/" -f 2
+```
+Output should be `amazon-k8s-cni:1.2.0` or newer. To upgrade to version 1.2, run the following command.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v1.2/aws-k8s-cni.yaml
+```
+
+## Follow Normal Instructions to Install
+
+Continue to [Installing Agones](#installing-agones).
+
+## Add HostPort to fleet or gameserver instructions
+
+After your installation is complete, add the following lines to any gameserver of fleet that you create to allow traffic to reach the container from the public IPv4 address.
+
+```
+apiVersion: "stable.agones.dev/v1alpha1"
+kind: GameServer
+metadata:
+  name: "simple-udp"
+spec:
+  ports:
+  - name: default
+    portPolicy: "dynamic"
+    containerPort: 7654             # ADD THIS
+    hostPort: 7654                  # ADD THIS
```

Comment:
Why do you need to set the hostPort, when the port policy is dynamic?

Also, you shouldn't have to set it at the pod level either - Agones should take care of that for you -- was there issue without it?

Reply:
Good catch, I set it since in my case it's portPolicy: "static"
No, I haven't tested it only at the pod level, let me try that now and verify it works.